packages/frontend-plugin-api/api-report.md

```diff
@@ -412,6 +416,17 @@ export interface ExtensionOverridesOptions {
   extensions: Extension<unknown>[];
 }
 
+// @public (undocumented)
+export function ExtensionSuspense(
```

Comment:
Thinking we could make this part of the extension boundary instead? Either via a `suspencs` or a `noSuspense`, or something like it

Reply:
Done, I think always using the suspense should work as well:
https://github.com/backstage/backstage/pull/20586/commits/66af240ea692929df76d8f817e96595d648c44d6